.github/workflows/tests.yml

```diff
@@ -34,24 +34,6 @@ jobs:
           command: test
           args: --all --lib -- --nocapture
 
-  test-ffi:
```

Comment:
We probably will need to write a similar test, which induces panic and checks if we handle it in libtelio error events somewhere in UniFFI part of things.

@mathiaspeters Could you add this as one of the "things to do" into the UniFFI tests task, so we do not forget about it?

Reply:
Done in [LLT-3475] ticket